src/docs/cookbook/design/snackbars.md

```diff
@@ -56,7 +56,7 @@ First, create a `SnackBar`, then display it using the `Scaffold`.
 final snackBar = SnackBar(content: Text('Yay! A SnackBar!'));
 
 // Find the Scaffold in the widget tree and use it to show a SnackBar.
```

Comment:
For example here and elsewhere this reference needs to be updated.
```suggestion
// Find the ScaffoldMessenger in the widget tree and use it to show a SnackBar.
```

It would also be a good idea to mention the ScaffoldMessenger, what it is, and that the one we are calling on here is the default ScaffoldMessenger provided by the MaterialApp.